quickwit/quickwit-doc-mapper/src/doc_mapping.rs

```diff
@@ -99,6 +100,10 @@ impl Default for Mode {
 #[derive(Clone, Debug, PartialEq, Serialize, Deserialize, utoipa::ToSchema)]
 #[serde(deny_unknown_fields)]
 pub struct DocMapping {
+    /// UID of the doc mapping.
+    #[serde(default = "DocMappingUid::new")]
```

Comment:
This is going to cause an issue when deserializing doc mapping without an UID (we want to them to be 0). I'm working on a fix.